pkg/utils/utils.go

```diff
@@ -374,6 +383,17 @@ func configSriovDevice(iface *sriovnetworkv1.Interface, ifaceStatus *sriovnetwor
 	return nil
 }
 
+func generateRandomMacAddressVfs() (string, error) {
+	buf := make([]byte, 6)
+	_, err := rand.Read(buf)
+	if err != nil {
+		return nil, err
+	}
+	// Set the local bit
+	buf[0] |= 2
+	return fmt.Sprintf("%02x:%02x:%02x:%02x:%02x:%02x", buf[0], buf[1], buf[2], buf[3], buf[4], buf[5]), nil
```

Comment:
Let's add a check and retry until we stop being unlucky (i.e. retry when we generate 00:00... or FF:FF... randomly).